### ISSUE TITLE
Gripper runtime - GripperClickOn

### ISSUE THREAD

**Issue by skull132** [Aurorastation/Aurora.3#1562]
@NanakoAC 
```
runtime error: 
[08:42:57]Cannot modify null.force.
[08:42:57]proc name: GripperClickOn (/mob/living/silicon/robot/proc/GripperClickOn)
[08:42:57]  source file: cyborg.dm,145
[08:42:57]  usr: (src)
[08:42:57]  src: Candy (/mob/living/silicon/robot)
[08:42:57]  src.loc: the floor (151,132,1) (/turf/simulated/floor/tiled)
[08:42:57]  call stack:
[08:42:57]Candy (/mob/living/silicon/robot): GripperClickOn(the oven (/obj/machinery/cooker/oven), "icon-x=16;icon-y=24;left=1;scr...", the service gripper (/obj/item/weapon/gripper/service))
[08:42:57]Candy (/mob/living/silicon/robot): ClickOn(the oven (/obj/machinery/cooker/oven), "icon-x=16;icon-y=24;left=1;scr...")
[08:42:57]the oven (/obj/machinery/cooker/oven): Click(the floor (153,138,1) (/turf/simulated/floor/tiled), "mapwindow.map", "icon-x=16;icon-y=24;left=1;scr...")
```

**Comment by LordFowl** [comment]
`[18:28:36] Process mob caught exception processing /mob/living/silicon/robot (Engineering Cyborg-651) (142,140,1): Cannot read null.loc in gripper.dm, line 118
This exception will now be ignored for ten minutes.`

`runtime error: 
[18:30:12]Cannot read null.loc
[18:30:12]proc name: drop (/obj/item/weapon/gripper/proc/drop)
[18:30:12]  source file: gripper.dm,118
[18:30:12]  usr: Engineering Cyborg-651 (/mob/living/silicon/robot)
[18:30:12]  src: the magnetic gripper (/obj/item/weapon/gripper)
[18:30:12]  usr.loc: the plating (142,140,1) (/turf/simulated/floor/plating)
[18:30:12]  src.loc: the engineering robot module (/obj/item/weapon/robot_module/engineering/general)
[18:30:12]  call stack:
[18:30:12]the magnetic gripper (/obj/item/weapon/gripper): drop(the plating (142,140,1) (/turf/simulated/floor/plating))
[18:30:12]the magnetic gripper (/obj/item/weapon/gripper): Drop Item()`

Two nasty runtimes. One or both of these actually make cyborgs/drones completely immortal.

**Comment by NanakoAC** [comment]
fixed by #1575

**Comment by NanakoAC** [comment]
hmm, i didnt notice there was another one, ill fix that too
